### DISCUSSION TITLE
broken page at github-pages - though i have added index.html in the root level of the branch

### DISCUSSION THREAD

**Discussion by fsj-digital** [community/community#36382] [Other Feature Feedback, Questions, & Ideas]
hi there

i have a brocken page - and i do not know why it is broken !? 

![image](https://user-images.githubusercontent.com/32641183/196126757-b1395016-9f56-4fa7-9679-7215e8443ff0.png)

well i tried to Make sure that i have a file named 'index.html' (all lower case) or '[README.md](https://readme.md/)' in the root level of the branch i selected as the publishing source for the  repository.

![image](https://user-images.githubusercontent.com/32641183/196127999-eedb6889-06d5-43e6-b32a-f94b835e5856.png)

at the moment i do not know why the page is broken!`?

**Comment by airtower-luna** [comment]
The red X next to the commit ID indicates that the build or deployment failed. Click on that and check the logs of the failed jobs to figure out what's wrong.

It'd be easier to help if you posted a link instead of a screenshot. :wink:

**Comment by fsj-digital** [comment]
hello dear @airtower-luna 
first of all : many many thanks for the quick reply - great to hear from you.  Well here some links

see
https://github.com/fsj-digital/DigitalHub
https://fsj-digital.github.io/DigitalHub/


![image](https://user-images.githubusercontent.com/32641183/196136637-05f3af86-3282-4244-a345-b33db8c808ae.png)


**btw**:  have tried to create a new page -  but all the page-creation failed here too: 

https://github.com/fsj-digital/digital_hubs/settings/pages
https://github.com/fsj-digital/digital_hubs/settings/pages

all and any help is greatly appreciated

**Comment by airtower-luna** [comment]
Your resume is here: https://fsj-digital.github.io/DigitalHub/resume/

The reason is that the YAML metadata in the README says it's supposed to go there: 

```yaml
permalink: /resume/
```

If you want to have the README as the top level page, removing that line should be enough. See also: https://jekyllrb.com/docs/permalinks/

**Comment by fsj-digital** [comment]
Hi there - awesome - this is just overwhelmiing . you solved the issue and saved my day.
